docs/contributing.md

```diff
@@ -47,4 +47,15 @@ Run `bash scripts/format-codes.sh` to format your codes.
 
 All submissions, including submissions by project members, require review. We
 use [GitHub pull requests](https://docs.github.com/articles/about-pull-requests)
-for this purpose.
\ No newline at end of file
+for this purpose.
+
+### Testing changes locally
```

Comment:
Nit: `Testing changes locally` to `Testing Changes Locally` to match the format.